ofl/kreon/METADATA.pb

```diff
@@ -1,34 +1,23 @@
 name: "Kreon"
 designer: "Julia Petretta"
 license: "OFL"
-category: "SERIF"
-date_added: "2011-02-02"
+category: "SANS_SERIF"
```

Comment:
Category should be SERIF and date should be the same.

Reply:
Ok, this has been updated in commit: b100e09